overview.md

```diff
@@ -0,0 +1,180 @@
+# STAC Overview
+
+There are three component specifications that together make up the core SpatioTemporal Asset Catalog specification.
+Each can be used alone, but they work best in concert with one another. The [STAC API specification](https://github.com/radiantearth/stac-api-spec) 
+builds on top of that core, but is out of scope for this overview. Items are the core atomic unit, representing a 
+single spatiotemporal asset as GeoJSON so it can be searched. Catalogs are structural elements, used to group Items
+and Collections. Collections *are* catalogs, that add more required metadata and describe a group of related Items.
```

Comment:
the "and Collections" at the end is a bit unexpected here...

Reply:
You're just saying I should introduce collections before saying catalogs are used to group them?

Reply:
Nevermind, I mis-read it yesterday. Probably due to the line-break I read it like "Catalogs are structural elements, used to group Items (comma) and Collections". Just realized how it really should read. It totally makes sense now ;-)